src/containers/contactsPage/ContactsPage.js

```diff
@@ -1,24 +1,31 @@
-import React, {useState, useEffect} from "react";
-import {ContactForm} from '../../components/contactForm';
-import {TileList} from '../../components/tileList'
+import React, { useState, useEffect } from "react";
+import { ContactForm } from '../../components/contactForm/ContactForm';
+import { TileList } from '../../components/tileList/TileList';
 
-export const ContactsPage = ({contactsArray, addContact}) => {
+export const ContactsPage = ({ contactsArray, addContact }) => {
   /*
   Define state variables for 
   contact info and duplicate check
   */
-const [name, setName] = useState('');
-const [phoneNumber, setPhoneNumber] = useState('');
-const [email, setEmail] = useState('');
-const [duplicateName, setDuplicateName] = useState(false);
+  const [name, setName] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [email, setEmail] = useState('');
+  const [duplicateName, setDuplicateName] = useState(false);
 
   const handleSubmit = (e) => {
```

Comment:
Is this appropriate?
`    if (isDuplicate) {
      setName("");
      return;
    } else {
      addContact(name, phoneNumber, email);
      setName("");
      setPhoneNumber("");
      setEmail("");
    }`

Should I reset all state variables if the form is submitted with it being a duplicate rather than just the name?

Reply:
Code is 👍 

Regarding the reseting the form when there is an error, there are many ways to go about this. 
I don't think any of the values should be set to blank. I think it should scroll to the top error to make the user aware again of the error. I know in this case its ok because the form is short, but you know when you have a super long form and you hit submit and don't realise why it isn't submitting, it can be annoying. You could add another list of errors below the submit button for when they try to submit.
This is more a user experience question so entirely up to you how much you want to optimise for this.